package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "test": "npm run lint && make && phantomjs node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js test/index.html dot",
     "start": "rollup --config=./rollup.config.js -w --input ./modules/id.js --output dist/iD.js",
+    "web": "node_modules/http-server/bin/http-server",
```

Comment:
`"web": "http-server",`

Should be sufficient. Node scripts have the local `node_modules/.bin/` automatically in their path